relay.yaml

```diff
@@ -0,0 +1,121 @@
+# This is the default configuration file for the relay.
+#
+# In addition to standard YAML syntax, you can use ${VAR} to substitute in the
+# value of the environment variable VAR, and ${VAR:DEFAULT} to provide a default
+# value if the variable is unset or empty. Many environment variable
+# substitutions have been provided by default to make it easy to configure the
+# relay for simple situations.
+#
+# When the value of an environment variable is substituted, it's converted to a
+# YAML primitive. This is almost always what you want, but in rare situations
+# where you need more control you can use $(VAR) or $(VAR:DEFAULT) to substitute
+# in the original, raw values.
+
+relay:
+  # The port on which the relay service should run.
+  port: ${RELAY_PORT:8990}
+
+  # The target to which traffic should be relayed, expressed as a URL-like
+  # scheme and host - e.g. "https://relay-target.example".
+  target: ${TRAFFIC_RELAY_TARGET}
+
+  # The maximum length in bytes which should be allowed for relayed response
+  # bodies. The default is 2MiB.
+  max-body-size: ${TRAFFIC_RELAY_MAX_BODY_SIZE:2097152}
+
+block-content:
+  # The 'body' option allows you to block content from request bodies. It
+  # contains a list of objects, each of which has either an 'exclude' property
+  # or a 'mask' property. The value of the property is a regular expression. For
+  # 'exclude', content matching the regular expression will be completely
+  # removed from the request body. For 'mask', matching content will be replaced
+  # with asterisks.
+  # Example:
+  # body:
+  #   - exclude: '\$[0-9]+(\.[0-9][0-9])?'  # Dollar quantities
+  #   - exclude: 'EXCLUDE ME'
```

Comment:
not directly related to the PR but should we include some form of option to rewrite excluded content to something other than empty string? for forensic purposes it's hard to determine if something was excluded to end up at the result; i.e.

```
body:
  - exclude: 'SECRET'
```

input: `fooSECRETbar`, output: `foobar` (current unchangeable behavior)
whereas if we allowed an arbitrary token, say `--excluded--` to be specified, we would end up with `foo--excluded--bar`, which you then know for sure it was excluded

Reply:
I agree that'd be useful. Happy to add that as a followup PR!